Dockerfile

```diff
@@ -57,18 +57,18 @@ FROM --platform=$BUILDPLATFORM golang:${GO_VERSION}-bookworm AS build-base-debia
 COPY --from=xx / /
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update -qq && apt-get install -qq --no-install-recommends \
-    git \
-    dpkg-dev
+  git \
```

Comment:
?

Reply:
ah probably editor auto format 🤦, let me change this back (and remove the 1.6.1 as well in the other comment), sry!

Reply:
@AkihiroSuda actually this file has mixed 2-space and 4-space indent. After this all indents are 2-space. Maybe we should keep this format change? 

4 spaces:
https://github.com/containerd/nerdctl/blob/1220ce7ec2701d485a9b1beeea63dae3da134fb5/Dockerfile#L65-L71

2 spaces:
https://github.com/containerd/nerdctl/blob/1220ce7ec2701d485a9b1beeea63dae3da134fb5/Dockerfile#L79-L83

Reply:
Either is fine to me.

Probably we should have some linter to enforce either of one.

Reply:
Sounds good, I will keep this format change for now then for consistency. Let me see if there are some linter that can run in CI.